# 4.15 Exclusive Relationships

In [9]:
import math
""" Confidence Interval of 90%  Function

pe = the population point estimate
std = standard deviation of the population
sample_size = size of the sample

"""
#Standard error method that comes into computation
def standard_error(std, sample_size):
    return (std/(math.sqrt(sample_size)))


def ci_90(pe, std,sample_size):
    z = 1.645

    se = standard_error(std,sample_size)
    ci_minus = pe - (z * se)
    ci_plus = pe + (z * se)
    ci = (ci_minus,ci_plus)
    #returns a tuple
    return ci

**Given Data :** 
<br/>
Sample Size = 203<br/>
Sample Average = 3.2<br/>
Standard Deviation = 1.97

Creating a 90 % confidence interval of the average number of exclusive relationships at Duke University

In [10]:
ci_90(3.2,1.97,203)

(2.9725509592882626, 3.4274490407117377)

**<u>Inference : </u> : ** We are 90% confident that undergraduate students have
been in 2.97 to 3.43 exclusive relationships, on average.